client/python/Dockerfile.redhat

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG BASE_IMAGE=registry.access.redhat.com/ubi8/ubi:8.4
+ARG FINAL_IMAGE=registry.access.redhat.com/ubi8/ubi-minimal:8.4
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
+FROM $BASE_IMAGE as base_build
+
+RUN groupadd --gid 5000 ovms && \
+    useradd --home-dir /home/ovms --create-home --uid 5000 --gid 5000 --shell /bin/bash --skel /dev/null ovms
+
+RUN dnf install -y unzip && \
+    curl -LO https://github.com/protocolbuffers/protobuf/releases/download/v3.18.0/protoc-3.18.0-linux-x86_64.zip && \
+    unzip protoc-3.18.0-linux-x86_64.zip -d /opt/protoc
+
+ENV PATH=$PATH:/opt/protoc
+
+RUN mkdir /ovmsclient
+COPY lib /ovmsclient/lib
+
+RUN dnf install -y python38 make && \
+    cd /ovmsclient/lib && make build
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
+FROM $FINAL_IMAGE as final_build
+
+ENV OVMSCLIENT_DIR="/home/ovmsclient"
+
+RUN microdnf install shadow-utils && \
+    groupadd --gid 5001 ovmsclient && \
+    useradd --home-dir ${OVMSCLIENT_DIR} --create-home --uid 5001 --gid 5001 --shell /bin/bash --skel /dev/null ovmsclient && \
+    microdnf remove shadow-utils && microdnf clean all
+
+COPY --from=base_build /ovmsclient/lib/dist/ovmsclient-0.1-py3-none-any.whl ${OVMSCLIENT_DIR}/ovmsclient-0.1-py3-none-any.whl
+COPY samples ${OVMSCLIENT_DIR}/samples
+
+RUN microdnf install -y python38 && microdnf clean all
+
+USER ovmsclient
+
+ENV PATH=${PATH}:${OVMSCLIENT_DIR}/.local/bin
+RUN pip3 install --user ${OVMSCLIENT_DIR}/ovmsclient-0.1-py3-none-any.whl && \
+    rm  ${OVMSCLIENT_DIR}/ovmsclient-0.1-py3-none-any.whl
+
+ENV PS1="$(whoami)@$(cat /etc/hostname):$(pwd)\$ "
```

Comment:
why do we need it?

Reply:
It's for better user experience. So when you enter the container in bash you have nicer prompt:
`<user>@<container_id>:<current_path>$`, for example
`ovmslient@1d34fsa154cs:/ovmsclient/samples $` 
instead of just something like:
`bash 4.4 $`